prime-dotnet-webapi/Migrations/20221025223517_SelfDeclarationUpdate.cs

```diff
@@ -0,0 +1,104 @@
+                    { 7, new DateTimeOffset(new DateTime(2019, 9, 16, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, -7, 0, 0, 0)), new Guid("00000000-0000-0000-0000-000000000000"), new DateTimeOffset(new DateTime(2022, 12, 1, 8, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 0, 0, 0, 0)), 3, "Has your access to PharmaNet or any other health information system, whether or not electronic and whether or not in British Columbia or another jurisdiction, been suspended or cancelled for a matter involving an “unlawful or improper action”?", new DateTimeOffset(new DateTime(2019, 9, 16, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, -7, 0, 0, 0)), new Guid("00000000-0000-0000-0000-000000000000") },
+                    { 8, new DateTimeOffset(new DateTime(2019, 9, 16, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, -7, 0, 0, 0)), new Guid("00000000-0000-0000-0000-000000000000"), new DateTimeOffset(new DateTime(2022, 12, 1, 8, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 0, 0, 0, 0)), 4, "Has an employer ever disciplined you, or terminated your employment, for a matter involving an “unlawful or improper action”?  Has a contract for your services ever been terminated for a matter involving an “unlawful or improper action”?", new DateTimeOffset(new DateTime(2019, 9, 16, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, -7, 0, 0, 0)), new Guid("00000000-0000-0000-0000-000000000000") }
+                });
+
+            migrationBuilder.CreateIndex(
+                name: "IX_SelfDeclaration_SelfDeclarationVersionId",
+                table: "SelfDeclaration",
+                column: "SelfDeclarationVersionId");
+
+            migrationBuilder.CreateIndex(
+                name: "IX_SelfDeclarationVersion_SelfDeclarationTypeCode",
+                table: "SelfDeclarationVersion",
+                column: "SelfDeclarationTypeCode");
+
+            migrationBuilder.AddForeignKey(
+                name: "FK_SelfDeclaration_SelfDeclarationVersion_SelfDeclarationVersi~",
+                table: "SelfDeclaration",
+                column: "SelfDeclarationVersionId",
+                principalTable: "SelfDeclarationVersion",
+                principalColumn: "Id",
+                onDelete: ReferentialAction.Restrict);
+        }
```

Comment:
I think it's possible to execute the script you included in the Jira item within the `Up` method, using `migrationBuilder.Sql`: 
 https://stackoverflow.com/questions/46638380/execute-custom-sql-script-as-part-of-entity-framework-migration

Reply:
Yes, I found some example in the previous migration as well. Add the sql to the up method. Thank you for the suggestion!